src/bin/pg_autoctl/pidfile.c

```diff
@@ -499,47 +499,59 @@ pidfile_as_json(JSON_Value *js, const char *pidfile, bool includeStatus)
 }
 
 
+bool
+is_process_stopped(const char *pidfile, bool *stopped, pid_t *pid)
```

Comment:
Missing function level header. Also, as before, what about having `pidfile_process_is_running` instead?

Reply:
Is discussed over chat, flipping this brings in other consistency problems. Leaving it like this.